### PR TITLE
Introduction to K8s

### PR DESCRIPTION
**Pull request by RaymondZhang24**

### CODE REVIEW

File: Topics/Introduction_To_Kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    - [Kubernetes vs Docker](#k8s-vs-docker)
+    - [Why k8s?](#why-k8s)
+2. [CI/CD](#CI/CD)
+3. [Examples](#Examples)
+
+## Introduction
+### What is Kubernetes(k8s)?
+
+Kubernetes known as k8s is a cluster manage tool and distrubuted supporting platform to use automate, deploying, and scaling container.
+
```
Comment (langsonzhang) [inline Topics/Introduction_To_Kubernetes.md:15]: Maybe briefly explain what containers are to someone who is unfamiliar with the technology

File: Topics/Introduction_To_Kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+### CI/CD
+k8s with CI/CD practices revolutionizes the software development lifecycle, offering agility and efficiency.  By integrating K8s with CI/CD pipelines, user can achieve faster release cycles, improved quality assurance, and greater overall productivity. CI/CD pipelines trigger automated tests and builds whenever new code is committed, with K8s enabling the seamless deployment of these changes into production environments. This integration ensures consistency between development, testing, and production environments, reducing the risk of errors and ensuring a smoother deployment process. With K8s and CI/CD, organizations can embrace a DevOps culture, delivering value to customers more quickly and efficiently while maintaining a high standard of readbilty and scalability in their applications.
+
+### Examples
+
+k8s are usually config in yaml format
+
```
Comment (langsonzhang) [inline Topics/Introduction_To_Kubernetes.md:33]: I think a high level overview of what each section of the config does could be a good addition

**Review by langsonzhang — COMMENTED** [review]
Added some comments

File: Topics/Introduction_To_Kubernetes.md (hunk 1 above)
Comment (RaymondZhang24) [reply Topics/Introduction_To_Kubernetes.md:15]: is explained in the docker section

**Review by WilliamJiH — APPROVED** [review]
This document provides a clear and concise overview of Kubernetes (k8s), effectively distinguishing between Kubernetes and Docker, highlighting their complementary roles in the container ecosystem. It emphasizes Kubernetes' strengths in managing containerized applications at scale, enhancing cloud-native computing with features like automated scaling, self-healing, and seamless integration with CI/CD pipelines for agile development and deployment. The examples section, with snippets for Deployment, Secrets, ConfigMaps, and CronJobs, offers practical insights into Kubernetes configuration, showcasing its ability to manage application deployment and scheduling tasks efficiently. However, the document could further benefit from a brief discussion on Kubernetes' networking and storage capabilities to provide a more comprehensive overview of its functionality.

**Pull request merged**